lib/logstash/filters/elasticsearch/dsl_executor.rb

```diff
@@ -0,0 +1,121 @@
+# encoding: utf-8
+
+module LogStash
+  module Filters
+    class Elasticsearch
+      class DslExecutor
```

Comment:
Just copied and pasted from without changing (only namings are standardized). This class may be simplified/cleanified for better maintenance.